libs/damap/src/lib/components/dashboard/dashboard.component.html

```diff
@@ -38,7 +42,13 @@ <h2 translate>dashboard.dmp.heading</h2>
         >{{ "dashboard.dmp.text.rdmPolicy.body" | translate }}
 
         {{ "dashboard.dmp.text.rdm.prefix" | translate }}
-        <a [href]="'dashboard.dmp.text.rdm.link' | translate" target="_blank">
+        <a
+          class="info-link-custom"
+          class="info-link-custom"
+          class="info-link-custom"
+          class="info-link-custom"
```

Comment:
Duplicate

Reply:
Duplicated class attribute (see also other components)

Reply:
ah yes, ctrl+D did add many of them. Ill clean this up, thanks for noticing!